docs/overview/blockchain-basics.md

```diff
@@ -0,0 +1,75 @@
+
+---
+## What is Web3?
+Web3 refers to the vision of a new, decentralized iteration of the internet that is built on blockchain technology and decentralized protocols. It represents a paradigm shift from the current centralized web (Web 2.0) to a more open, transparent, and user-centric internet ecosystem.
+
+Overall, Web3 represents a vision for a more decentralized, user-centric internet that fosters innovation, transparency, and financial inclusivity. It aims to address some of the limitations and challenges of the current centralized web, such as data privacy concerns, censorship, and lack of user control.
+
+---
+## Why do we need decentralization?
+Decentralization is essential because it provides resilience against single points of failure, ensuring that no single entity or authority has absolute control. It fosters censorship resistance, enabling freedom of expression and financial autonomy.
+
+Moreover, decentralization enhances security by distributing data across a network, making it harder for malicious actors to compromise. It also promotes innovation by lowering barriers to entry and fostering competition among participants.
+
+Additionally, decentralization empowers individuals by giving them greater control over their personal data and financial assets, promoting financial inclusivity and transparency in governance and decision-making processes.
+
+---
+## Why Koinos?
+Koinos is engineered from the ground up to be energy efficient, promote decentralization, infinitely evolve, execute extremely quickly, and most of all allow for transactions without fees. The unique design of Koinos allows developers to create unstoppable decentralized applications with no barrier to onboard new users.
+
+### For developers
+It is core to Koinos to focus on the developer experience. This is why it is designed in a language agnostic manner allowing developers to work with programming languages developers know and love.
+
+The inherent problem of transaction fees has plagued smart contracts platforms for far too long. It is now possible to give end users a Web2 experience on the next generation Web3 platform, Koinos.
+
+Don't kneecap your vision, shoot for the stars and onboard the masses on Koinos!
+
+### For users
+For end users, the ability to interact with their favorite decentralized applications without transaction fees offers a user experience like no other blockchain and one that can finally rival Web2.
+
+Play Web3 games where you control your own in-game assets, manage your financial future with DeFi, be free of censorship all without paying a dime in transaction fees!
+
+---
+## Join the revolution
+You're not alone! Join our growing community of dedicated blockchain developers, users, and enthusiasts on both [Telegram](https://t.me/joinchat/SXe6mLnb8Ssywlq9) and [Discord](https://discord.com/invite/GErGNsu) today!
```

Comment:
We have discord.koinos.io redirecting to the discord invite link. We should be using it wherever possible in case the invite link ever needs to be updated.

Reply:
Definitely. It would be nice to have one of these for Telegram too.

I got these links from koinos.io so I supposed we should update it there too.

Reply:
telegram.koinos.io is now an active redirect.